2 - ListsLogicAndIterations/Code/Arrays.cs

```diff
@@ -6,12 +6,18 @@ public static class Arrays
 	{
 		public static int[] BuildIntArray(int fist, int second)
 		{
-			throw new NotImplementedException();
+            int[] returnArray = new int[2] { fist, second };
+            return returnArray;
```

Comment:
This totally works but you can make the code a bit more simplified by doing:

`return new int[] { first, second };`

This because when defining the array if you don't specify a count it will determine it based on the amount of elements you add to it for you. Helps for refactoring later too.